packages/website/package.json

```diff
@@ -35,32 +35,21 @@
     "@typescript-eslint/eslint-plugin": "^2.33.0",
     "@typescript-eslint/parser": "^2.34.0",
     "@vue/cli": "^4.3.1",
-    "@vue/cli-plugin-babel": "^4.3.1",
     "@vue/cli-plugin-eslint": "^4.3.1",
     "@vue/cli-plugin-typescript": "^4.3.1",
     "@vue/cli-service": "^4.3.1",
-    "@vue/component-compiler-utils": "^3.1.2",
     "@vue/eslint-config-airbnb": "^5.0.2",
     "@vue/eslint-config-prettier": "^6.0.0",
     "@vue/eslint-config-typescript": "^5.0.1",
-    "autoprefixer": "^9.8.0",
-    "babel-plugin-transform-vue-jsx": "^3.7.0",
```

Comment:
This is needed for TSX support.

Reply:
`@vue/babel-preset-jsx` is added instead since it has official vue support and it doesn't require you to write a long config in `vue.config.js`

Reply:
The long config was exported from `vue-cli` and modified, because the TypeScript plugin didn't support loading TSX when this was originally built. So while the preset has always worked for loading JSX, it used to fail when loading TSX as the TypeScript plugin choked on TSX.

https://github.com/vuejs/vue-cli/pull/4894 starts the work fixing the TSX support.